lecture_2/homework/db/schema.rb

```diff
@@ -10,18 +10,52 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_03_14_150921) do
+ActiveRecord::Schema.define(version: 2019_03_28_190612) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "castles", force: :cascade do |t|
+  end
+
   create_table "clans", force: :cascade do |t|
     t.string "name", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.index ["name"], name: "index_clans_on_name", unique: true
+  end
+
+  create_table "gates", force: :cascade do |t|
+  end
+
+  create_table "magicals", force: :cascade do |t|
```

Comment:
W przypadku używania STI nie ma potrzeby tworzenia tabel dla każdego z tych modeli. Zgodnie z nazwą jest to "Single Table Inheritance", więc wymaga tylko jednej tabeli - w tym przypadku "weapons"